xamarin/Xamarin.Forms
# [Android] Fixed memory leak applying brushes to shapes

### Description of Change ###

Fixed memory leak applying brushes to shapes on Android.

### Issues Resolved ### 

- fixes #11957 

### API Changes ###
 
 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
Launch Core Gallery with the Profiler, and navigate to the issue 11957. Verify that the memory is not being increased unnecessarily.

### PR Checklist ###
